RecoTracker/IterativeTracking/python/PixelLessStep_cff.py

```diff
@@ -288,14 +292,20 @@
 
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(pixelLessStep, pixelLessStepClassifier1.clone(
+	mva = dict(GBRForestLabel = 'MVASelectorPixelLessStep_Phase1'),
+	src = 'pixelLessStepTracks',
```

Comment:
this src= is redundant and was not present in the pre-DNN version.

Reply:
Removed redundant line